pinot-common/src/main/java/org/apache/pinot/common/function/FunctionRegistry.java

```diff
@@ -205,5 +205,15 @@ public static boolean textMatch(String text, String pattern) {
     public static boolean jsonMatch(String text, String pattern) {
       throw new UnsupportedOperationException("Placeholder scalar function, should not reach here");
     }
+
+    @ScalarFunction(names = {"split", "split"}, isPlaceholder = true)
```

Comment:
With this change, we can handle the `split` function in multistage. 

The other option available is to leave it as it is i.e. returning `string[]`. In that case, it will translate to `SqlTypeName.OTHER` and eventually become `DataSchema.ColumnDataType.OBJECT`. To get it working we need to add support in `ObjectSerDeUtils` for `string[]`.

Currently, it's following the above-mentioned option and failing with an exception getting generated from `ObjectSerDeUtils` 
```
throw new IllegalArgumentException("Unsupported type of value: " + value.getClass().getSimpleName());
```

Reply:
Gotcha, let me revert the `split` changes.